### PR TITLE
Fixed commented test cases of all Files

### PR DESCRIPTION
**Pull request by Himanshu1231231**

### CODE REVIEW

**Review by ishubham1995 — APPROVED** [review]
LGTM

File: cypress/integration/api/tests/CatalogPageTest.js [hunk 1]
```diff
@@ -363,13 +365,15 @@ describe("Catalog Page", () => {
         })
     })
   
+    //**Error: 500 Internal Server Error */
```
Comment (TiwariMithilesh) [inline cypress/integration/api/tests/CatalogPageTest.js:368]: Please write an proper text, if this API is not available in Swagger just write a proper text.

File: cypress/integration/api/tests/CatalogPageTest.js [hunk 2]
```diff
@@ -363,13 +365,15 @@ describe("Catalog Page", () => {
...
     // it('Get Catalog Platform Custom Domain', () => {
     //     doGetCatalogPlatformCustomDomain(authKey, platform_id,app_id).then((response) => {
     //         expect(response.status).to.eq(200)
     //         cy.log("Get Catalog Platform Custom Domain", response.body)
     //     })
     // })
    
+    //** Error: 500 Internal Server Error*/
```
Comment (TiwariMithilesh) [inline cypress/integration/api/tests/CatalogPageTest.js:376]: Please write an proper text, if this API is not available in Swagger just write a proper text.

**Review by TiwariMithilesh — COMMENTED** [review]
Please write an proper text, if this API is not available in Swagger just write a proper text.

File: cypress/integration/api/tests/CatalogPageTest.js (hunk 1 above)
Comment (Himanshu1231231) [reply cypress/integration/api/tests/CatalogPageTest.js]: Resolved

File: cypress/integration/api/tests/CatalogPageTest.js (hunk 2 above)
Comment (Himanshu1231231) [reply cypress/integration/api/tests/CatalogPageTest.js]: Resolved

**Pull request merged**